examples/complete/example.yaml

```diff
@@ -0,0 +1,22 @@
+datadog_api_key: ENC[AES256_GCM,data:i3Scnwrz2EX87g==,iv:fHkcmMN3lVzxNVuQqdrTe6tV1gNGLdeb+Hks9CFC+3k=,tag:2+jWdi+WJO7iIOOwvjtHog==,type:int]
+datadog_app_key: ENC[AES256_GCM,data:mOBHdrNr4QrVgw==,iv:DCRSQgpnPflxGLIMKcuUTFBe2vrGadul5FoQ8Wn+fcE=,tag:6nfD4HFN8tPiu6iBU3fTTg==,type:int]
```

Comment:
Just to confirm, these are dummy values?

Reply:
Yep:
```yaml
datadog_api_key: 1234567890
datadog_app_key: 9876543210
```

Reply:
Good stuff -- Thank you!